css/drift.css

```diff
@@ -0,0 +1,1180 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
+/* Document
+   ========================================================================== */
+/**
+ * 1. Correct the line height in all browsers.
+ * 2. Prevent adjustments of font size after orientation changes in iOS.
+ */
```

Comment:
try to use css variables as much as possible. variables will give your component library uniformity

Reply:
It is compiled css, I'm using sass

Reply:
aaah ok. then its ok.

i have not used sass, i am guessing it is replacing your variables into values after compilation?

Reply:
Yes, correct✌ @shubham-singh